### PR TITLE
bug 1782900 MITM proxy settings

### PR DESCRIPTION
**Pull request by kalexand-rh**
https://bugzilla.redhat.com/show_bug.cgi?id=1782900

@jiajliu, will you PTAL?

### CODE REVIEW

**Comment by jiajliu** [comment]
@kalexand-rh 
It's not for all MITM proxy setting, and most of situations, httpproxy and httsproxy are also needed even for MITM proxy. It's only a workaround for a special scenario that when user's internet with a transparent proxy network with no additional proxy configuration needed, but additional CAs needed due to it's a MITM proxy. So it's better to update for this special scenario only to avoid misunderstanding.
More detail refer to https://bugzilla.redhat.com/show_bug.cgi?id=1771564#c0

**Comment by kalexand-rh** [comment]
@jiajliu, will you please take another look?

**Comment by jiajliu** [comment]
LGTM

**Comment by kalexand-rh** [comment]
Thank you @jiajliu!  @openshift/team-documentation, PTAL?

File: modules/installation-configure-proxy.adoc [hunk 1]
```diff
@@ -71,16 +71,17 @@ additionalTrustBundle: | <4>
 ...
 ----
 <1> A proxy URL to use for creating HTTP connections outside the cluster. The
-URL scheme must be `http`.
+URL scheme must be `http`. If you use an MITM transparent proxy network that does not require additional proxy configuration but requires additional CAs, you must not specify a `httpProxy` value.
```
Comment (bergerhoffer) [inline modules/installation-configure-proxy.adoc:74]: a `httpProxy` -> an `httpProxy`

File: modules/installation-configure-proxy.adoc [hunk 2]
```diff
@@ -71,16 +71,17 @@ additionalTrustBundle: | <4>
...
 <2> A proxy URL to use for creating HTTPS connections outside the cluster. If
 this field is not specified, then `httpProxy` is used for both HTTP and HTTPS
 connections. The URL scheme must be `http`; `https` is currently not
 supported.
+If you use an MITM transparent proxy network that does not require additional proxy configuration but requires additional CAs, you must not specify a `httpsProxy` value.
```
Comment (bergerhoffer) [inline modules/installation-configure-proxy.adoc:79]: a `httpsProxy` -> an `httpsProxy`

**Review by bergerhoffer — COMMENTED** [review]
Two minor suggestions, otherwise LGTM!

**Pull request merged**